script.js

```diff
@@ -9,4 +13,153 @@ hamburger.addEventListener('click', () => {
 document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
   hamburger.classList.remove('active');
   menu.classList.remove('active');
-}));
\ No newline at end of file
+}));
+
+const body = document.querySelector('body');
+const section = document.createElement('section');
+const cardGrid = document.querySelector('.works .works-list');
+const cardGridArticle = document.createElement('article');
+function implementProjects() {
+  myProjects.forEach((project) => {
+    const projectArticle = `
+    <div class="first-work">
+    <div class="image area-B">
+      <img
+        src="${project.projectImage}"
+        alt="multi-Post project picture"
+      />
+    </div>
+    <div class="info area-A">
+      <h2 class="info-tittle">${project.projectName}</h2>
+      <ul class="ul-info">
+        <li class="canopy">${project.companyName}</li>
+        <li>
+          <img src="images/Counter.svg" alt="counter icon" />
+        </li>
+        <li class="back-end">${project.projectPosition}</li>
+        <li>
+          <img src="images/Counter.svg" alt="counter icon" />
+        </li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="paragraph">
+        ${project.projectDescription}
+      </p>
+      <ul class="ul-languages" aria-label=" Programming languages">
+        <li>${project.projectTechHtml}</li>
```

Comment:
- [ ] Kindly work on the styling part of the pop-up on a desktop version. I have screenshots below to help you see what needs to be done.

## Figma

![popup](https://user-images.githubusercontent.com/53124548/209353287-7548da08-e1e8-4f3a-bff8-735b01133e56.png)

## Your design

![pop_yours](https://user-images.githubusercontent.com/53124548/209353395-4623b3d8-48ad-4abe-9352-2f135e5b7df8.png)